### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by roidelapluie**
Demo: https://github.com/roidelapluie/prometheus/issues/new/choose

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Users Mailing List
```
Comment (juliusv) [inline .github/ISSUE_TEMPLATE/config.yml]: I'd un-title-case this ("Users mailing list" and "Prometheus community" below) so it's consistent in casing with the first two entries ("Bug report" and "Feature request").

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
+---
+name: Bug report
+about: Create a report to help us improve.
+title: ''
+labels: kind/bug
+assignees: ''
+
```
Comment (juliusv) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Is there a reason for this extra empty line here and in the other file?

Comment (roidelapluie) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I generated the files with the Github Settings helper.

Comment (juliusv) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Ah ok, maybe it adds an extra empty line, oh well :man_shrugging:

**Comment by juliusv** [comment]
:+1: Thanks, really love it besides the nits!

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (roidelapluie) [reply .github/ISSUE_TEMPLATE/bug_report.md]: It works if I remove the line.

**Pull request merged**